app/controllers/api/internal/v2/court_application_laa_references_controller.rb

```diff
@@ -6,16 +6,28 @@ def create
           contract = CourtApplicationLaaReferenceContract.new.call(**transformed_params)
           enforce_contract!(contract)
 
-          enqueue_link
-          head :accepted
+          CourtApplicationMaatLinkCreator.call(
+            transformed_params[:subject_id],
+            transformed_params[:user_name],
+            transformed_params[:maat_reference],
+          )
+
+          head :created
         end
 
         def update
           contract = UnlinkCourtApplicationSubjectContract.new.call(**transformed_params)
           enforce_contract!(contract)
 
-          enqueue_unlink
-          head :accepted
+          CourtApplicationLaaReferenceUnlinker.call(
+            subject_id: transformed_params[:subject_id],
+            user_name: transformed_params[:user_name],
+            unlink_reason_code: transformed_params[:unlink_reason_code],
+            unlink_other_reason_text: transformed_params[:unlink_other_reason_text],
+            maat_reference: transformed_params[:maat_reference],
+          )
+
+          head :ok
```

Comment:
Mo's postman tests are still expecting 202s here - you'll need to update them to get them to pass.

Reply:
Update: I had to include the 404 status as well, since it is a sync, the not found is also a possible status now.